tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/GoLanguageSpecificChecks.cs

```diff
@@ -0,0 +1,163 @@
+        }
+    }
+    public async Task<CLICheckResponse> FormatCodeAsync(string packagePath, CancellationToken ct)
+    {
+        try
+        {
+            var result = await _processHelper.Run(new ProcessOptions(
+                formatterName, ["-w", "."],
+                formatterNameWindows, ["-w", "."],
+                workingDirectory: packagePath
+            ), ct);
+            return CLICheckResponse.CreateResponseFromProcessResult(result);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "{MethodName} failed with an exception", nameof(FormatCodeAsync));
+            return new CLICheckResponse(1, "", $"{nameof(FormatCodeAsync)} failed with an exception: {ex.Message}");
+        }
+    }
+
+    public async Task<CLICheckResponse> LintCodeAsync(string packagePath, CancellationToken ct)
+    {
+        try
+        {
+            var result = await _processHelper.Run(new ProcessOptions(linterName, ["run"], workingDirectory: packagePath), ct);
+            return CLICheckResponse.CreateResponseFromProcessResult(result);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "{MethodName} failed with an exception", nameof(LintCodeAsync));
+            return new CLICheckResponse(1, "", $"{nameof(LintCodeAsync)} failed with an exception: {ex.Message}");
+        }
+    }
+
+    public async Task<CLICheckResponse> RunTestsAsync(string packagePath, CancellationToken ct)
+    {
+        try
+        {
+            var result = await _processHelper.Run(new ProcessOptions(compilerName, ["test", "-v", "-timeout", "1h", "./..."], workingDirectory: packagePath), ct);
+            return CLICheckResponse.CreateResponseFromProcessResult(result);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "{MethodName} failed with an exception", nameof(RunTestsAsync));
+            return new CLICheckResponse(1, "", $"{nameof(RunTestsAsync)} failed with an exception: {ex.Message}");
+        }
+    }
+
+    public async Task<CLICheckResponse> BuildProjectAsync(string packagePath, CancellationToken ct)
+    {
+        try
+        {
+            var result = await _processHelper.Run(new ProcessOptions(compilerName, ["build"], workingDirectory: packagePath), ct);
+            return CLICheckResponse.CreateResponseFromProcessResult(result);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "{MethodName} failed with an exception", nameof(BuildProjectAsync));
+            return new CLICheckResponse(1, "", $"{nameof(BuildProjectAsync)} failed with an exception: {ex.Message}");
+        }
+    }
+
+    public string GetSDKPackagePath(string repo, string packagePath)
+    {
+        if (!repo.EndsWith(Path.DirectorySeparatorChar))
+        {
+            repo += Path.DirectorySeparatorChar;
+        }
+
+        // ex: sdk/messaging/azservicebus
```

Comment:
```suggestion
        // ex: sdk/messaging/azservicebus/
```

Given the above wouldn't it be this?